scripts/backend-lint.sh

```diff
@@ -16,25 +16,7 @@ go get -u github.com/jgautheron/goconst/cmd/goconst
 go get -u honnef.co/go/tools/cmd/staticcheck
 go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
 
-# use gometalinter when lints are not available in golangci or
-# when gometalinter is better. Eg. goconst for gometalinter does not lint test files
-# which is not desired.
-exit_if_fail gometalinter --enable-gc --vendor --deadline 10m --disable-all \
-  --enable=goconst\
-  --enable=staticcheck
-
-# use golangci-when possible
-exit_if_fail golangci-lint run --deadline 10m --disable-all \
-  --enable=deadcode\
-  --enable=gofmt\
-  --enable=gosimple\
-  --enable=govet\
-  --enable=ineffassign\
-  --enable=structcheck\
-  --enable=typecheck\
-  --enable=unconvert\
-  --enable=unused\
-  --enable=varcheck
+exit_if_fail make golangci-lint
```

Comment:
All the `go get ...` calls should be removed. No need to fetch these here when that's built into the Makefile.

Reply:
I was planning to tackle continuous of this later one, but might as well do it here :)